src/main/java/com/tokyoprogrammer/aqitracer/controller/AqitracerController.java

```diff
@@ -1,13 +1,47 @@
 package com.tokyoprogrammer.aqitracer.controller;
 
-import org.springframework.web.bind.annotation.GetMapping;
-import org.springframework.web.bind.annotation.RestController;
+import org.springframework.stereotype.Component;
 
-@RestController
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.HttpStatus;
+import com.linecorp.armeria.common.MediaType;
+import com.linecorp.armeria.internal.shaded.guava.collect.ImmutableMap;
+import com.linecorp.armeria.server.annotation.Get;
+import com.linecorp.armeria.server.annotation.Param;
+import com.linecorp.armeria.server.annotation.Path;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Component
+@Slf4j
 public class AqitracerController {
 
-    @GetMapping("/healthCheck")
-    public String healthCheck() {
-        return "OK";
+    private enum AQI_PROVIDER {
+        WAQI,
+        KECO,
+        EUROPE,
+    };
+
+    private enum ETC_TYPE {
+        WALK,
+        CULTURE
+    };
+
+    @Get
+    @Path("/healthCheck")
```

Comment:
Does Armeria server provide reactive endpoints like Spring WebFlux? I guess the Armeria annotations such as `@Get` and `@Path` are not so different from `@GetMapping`, am I correct? If not, could you explain more about it?

Reply:
As you said, Armeria supports the integration with Spring WebFlux. Please refer to it https://line.github.io/armeria/advanced-spring-webflux-integration.html?highlight=webflux I think we can consider reactive stream 👍